src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,55 @@
+package duke;
+
+import duke.command.Command;
+import duke.exception.DukeException;
+import duke.task.TaskList;
+import duke.ui.MessageUi;
+
+
+/**
+ * Represents the main class that Ekud will run on.
+ */
+public class Duke {
+
+    private MessageUi ui;
+    private Storage storage;
+    private TaskList tasks;
+    private boolean f;
+
+
+    public Duke() {
+    }
+
+    /**
+     *
+     * @param filePath Directory of text file.
+     * @throws DukeException
+     */
+    public Duke(String filePath) throws DukeException {
+        ui = new MessageUi();
+        storage = new Storage(filePath);
+        try {
+            tasks = new TaskList(storage.loadFileContents());
+            f = true;
+        } catch (DukeException e) {
+            tasks = new TaskList();
+        }
+    }
+
+    public String getFileLoadingMessage() {
```

Comment:
Would it be better to add documentation to describe the functionality of this function? 😃